fiat-web/fiat-web.gradle

```diff
@@ -48,4 +49,21 @@ dependencies {
   testImplementation "com.netflix.spinnaker.kork:kork-jedis-test"
 }
 
+test {
```

Comment:
Same as here: https://github.com/spinnaker/orca/blob/master/orca-web/orca-web.gradle#L104

Not great but needed for the plugins TCK to work